chainer/training/trainer.py

```diff
@@ -184,7 +185,10 @@ def extend(self, extension, name=None, trigger=None, priority=None,
             trigger (tuple or Trigger): Trigger object that determines when to
                 invoke the extension. If it is ``None``, ``extension.trigger``
                 is used instead. If the trigger is not callable, it is passed
-                to :class:`IntervalTrigger` to build an interval trigger.
+                to :class:`IntervalTrigger` to build an interval trigger. If
```

Comment:
I think the sentence begin with "If it is None and the extension ... as default" should be placed that after the sentence "If it is None, extension.trigger is used instead." because it would make the document more readable.